requirements.txt

```diff
@@ -1,6 +1,6 @@
 distlib==0.3.1
 filelock==3.0.12
-flask=1.1.2
+flask==1.1.2
```

Comment:
We shouldn't need to be keeping this up to date any more, in fact let's get rid of it since we're using poetry to manage all of our dependencies anyway